pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -78,3 +78,17 @@ type RemoveDuplicatesArgs struct {
 	Namespaces        *api.Namespaces
 	ExcludeOwnerKinds []string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// RemovePodsViolatingTopologySpreadConstraintArgs holds arguments used to configure RemovePodsViolatingTopologySpreadConstraint plugin.
+type RemovePodsViolatingTopologySpreadConstraintArgs struct {
+	metav1.TypeMeta
+
+	Namespaces                 *api.Namespaces
+	LabelSelector              *metav1.LabelSelector
+	IncludeSoftConstraints     bool
+	ThresholdPriority          *int32
+	ThresholdPriorityClassName string
+	NodeFit                    bool
```

Comment:
```suggestion
```

I'm pretty sure `NodeFit` is also no longer needed. Added to [EvictionFilter](https://github.com/kubernetes-sigs/descheduler/blob/83c4f5d5261e7113b6367c70914cedd1af7792ff/pkg/descheduler/descheduler.go#L388) here